pom.xml

```diff
@@ -53,6 +52,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.level>8</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError> <!-- TODO -->
```

Comment:
I would loved you possibly put a small explanatory comment as commit log in the [commit disabling FindBugs](https://github.com/jenkinsci/docker-fixtures/pull/22/commits/594a835fbee4768f968bff7c2ae395b1605279b6). What is failing?